### PR TITLE
Fixed build status badge.

### LINKED ISSUE #148: Change repo build status from Travis-CI to Github actions
In README.md, that status points to

`[Build Status](https://travis-ci.org/TDycores-Project/TDycore)`

libCEED has the following setup https://raw.githubusercontent.com/CEED/libCEED/main/README.rst

### PR DESCRIPTION
**Pull request by jeff-cohere**
Closes #148

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/TDycores-Project/TDycore.svg?branch=master)](https://travis-ci.org/TDycores-Project/TDycore)
+[![Build Status](https://github.com/TDycores-Project/TDycore/workflows/auto_pr_test/badge.svg)
```
Comment (jedbrown) [inline README.md:1]: How is this not malformed? Seems you need a link target.

Comment (jeff-cohere) [reply README.md:1]: Ah, right. The symbol soup and the long turnaround time got me. Will double check. Thanks!

Comment (jeff-cohere) [reply README.md]: Apparently it wasn't malformed--the first [ was just ignored, and the rest of the URI was correct. Too much to ask for consistent syntax in Markdown. Fixed now.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/TDycores-Project/TDycore.svg?branch=master)](https://travis-ci.org/TDycores-Project/TDycore)
+![Build Status](https://github.com/TDycores-Project/TDycore/workflows/auto_pr_test/badge.svg)
```
Comment (jedbrown) [inline README.md:1]: It should link to someplace useful.
```suggestion
[![Build Status](https://github.com/TDycores-Project/TDycore/workflows/auto_pr_test/badge.svg)](https://github.com/TDycores-Project/TDycore/actions)
```

**Review by jedbrown — APPROVED** [review]
Approved to apply the suggestion and squash-merge.

**Review by bishtgautam — APPROVED** [review]

**Pull request merged**